### PR TITLE
fixed create an assignment bug #709

### PR DESCRIPTION
**Pull request by goldenhub77**

### CODE REVIEW

File: app/controllers/assignments_controller.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def show
 
   def new
     @assignment = Assignment.new
+    @assignment.requirement = @requirement
```
Comment (kgf) [inline app/controllers/assignments_controller.rb:18]: Isn't this covered by line 5, the before hook ?

**Comment by kgf** [comment]
@goldenhub77 Thanks for your assistance. Only one comment.

Could you also PR this against v2.10.0 branch ? That's what's in production now

File: app/controllers/assignments_controller.rb (hunk 1 above)
Comment (goldenhub77) [reply app/controllers/assignments_controller.rb:18]: @kgf This is a necessary change. I had to change the forms and assignment needs to have a requirement set before it gets to the view or you will get no method for null errors while its loading the form.

Comment (kgf) [reply app/controllers/assignments_controller.rb:18]: Ah, I didn't parse this correctly the first time. The before hook **is** defining the requirement, and here you're making it the parent to the new assignment.

**Comment by kgf** [comment]
Thanks @goldenhub77

**Pull request merged**